studio/src/editors/merch-card-editor.js

```diff
@@ -1,37 +1,37 @@
-import { html, LitElement, css, nothing } from 'lit';
+import { html, nothing } from 'lit';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
 import '../fields/multifield.js';
 import '../fields/mnemonic-field.js';
+import '../aem/aem-tag-picker-field.js';
+import './variant-picker.js';
+import EditorPanel from '../editor-panel.js';
 
 const MODEL_PATH = '/conf/mas/settings/dam/cfm/models/card';
 
-class MerchCardEditor extends LitElement {
+class MerchCardEditor extends EditorPanel {
```

Comment:
I don't see why extending from EditorPanel is needed.

I think this also resulted in the following being displayed at the top of the editor instead of the bottom.
![image](https://github.com/user-attachments/assets/072226bd-5747-4db0-9db3-e736506f664e)

Reply:
editor-panel renders merch-card-editor, so the latter should not extends the former.

Reply:
Removed class inheritance